core/extensions/impl/crypto_extension.cpp

```diff
@@ -168,4 +174,75 @@ namespace kagome::extensions {
 
     memory_->storeBuffer(out_ptr, common::Buffer(hash));
   }
+
+  runtime::PointerSize CryptoExtension::ext_crypto_secp256k1_ecdsa_recover_v1(
+      runtime::WasmPointer sig, runtime::WasmPointer msg) {
+    static auto make_result = [&](auto &&value) {
```

Comment:
lambda used only once. Do we really need it?

Reply:
refactored